ibm_catalog.json

```diff
@@ -850,7 +895,7 @@
             },
             {
               "name": "deploy-arch-ibm-account-infra-base",
-              "description": "Cloud automation for Account Configuration organizes your IBM Cloud account with a ready-made set of resource groups by default and, when you enable the \"with Account Settings\" option, it also applies baseline security and governance settings. ",
+              "description": "Organize your IBM Cloud account with preconfigured resource groups. If not selected, the default group is used. Optionally, expand to apply recommended security settings via \"with Account Settings\".",
```

Comment:
`If not selected, the default resource group is used`

via \"with Account Settings\" variation.

Reply:
`recommended security controls`